src/options.zig

```diff
@@ -1529,6 +1529,9 @@ pub const BundleOptions = struct {
     /// So we have a list of packages which we know are safe to do this with.
     unwrap_commonjs_packages: []const string = &default_unwrap_commonjs_packages,
 
+    outfile: bool = false,
```

Comment:
outfile being a boolean is kind of silly, can we do without this?

Reply:
You're totally right, folded this into the variable supports_multiple_outputs